File: ITP1/ITP1_2_D_Circle_in_a_Rectangle.py
w,h,x,y,r=map(int,input().split())
print(("No","Yes")[(r<=x<=w-r)*(r<=y<=h-r)])
# W, H, x, y, r = map(int, input().split())
# print(["No", "Yes"][(r <= x <= W - r) * (r <= y <= H - r)])

File: AOJ/ITP1/ITP1_2_D_Circle_in_a_Rectangle.cs
using System;

class P
{
    static void Main()
    {
        var a = Array.ConvertAll(Console.ReadLine().Split(), int.Parse);
        int r = a[4];
        Console.WriteLine(a[2] < r || a[0] - r < a[2] || a[3] < r || a[1] - r < a[3] ? "No" : "Yes");
    }
}
